Remotion/Web/UnitTests/Core/UI/Controls/HtmlHeadContentsImplementation/Rendering/HtmlHeadContentsRendererTest.cs

```diff
@@ -46,9 +47,9 @@ public void Render_SortsWellknownElementsByType ()
       var htmlHelper = new HtmlHelper();
 
       var htmlHeadContentsRenderingContext = new HtmlHeadContentsRenderingContext (
-          MockRepository.GenerateStub<HttpContextBase>(),
+          new Mock<HttpContextBase>().Object,
```

Comment:
In such cases, instead of `new Mock<HttpContextBase>().Object`, `Mock.Of<HttpContextBase>()` could be written. This would be shorter but introduce an inconsistency in terms of mock instantiation.

Opinion @MichaelKetting ?